WORKSPACE

```diff
@@ -1,17 +1,5 @@
 workspace(name = "proto_gen_firebase_rules")
 
-protobuf_commit = "099d99759101c295244c24d8954ec85b8ac65ce3"
+load(":bazel/repositories.bzl", "protobuf_rules_gen_repositories")
```

Comment:
`//bazel:repositories.bzl` should work and be a more canonical style

Reply:
done